cmd/kubeapps-apis/server/plugins.go

```diff
@@ -54,16 +70,18 @@ func NewPluginsServer(pluginDirs []string, registrar grpc.ServiceRegistrar, gwAr
 		log.Fatalf("failed to check for plugins: %v", err)
 	}
 
-	pluginDetails, err := registerPlugins(pluginPaths, registrar, gwArgs)
+	ps := &pluginsServer{}
+
+	pluginDetails, err := ps.registerPlugins(pluginPaths, registrar, gwArgs)
```

Comment:
This file is refactored a little so that the plugin detail can be passed down the chain of calls and included together with the implementation server to use in the core server.